pgml-extension/src/bindings/transformers/mod.rs

```diff
@@ -52,6 +52,43 @@ pub fn transform(
     Ok(serde_json::from_str(&results)?)
 }
 
+pub fn transform_stream(
```

Comment:
Why not replace the current transform implementation with the streaming version?

Reply:
I guess if the user calls it without a limit or cursor, they do the same exact thing. I'm not sure what the overhead of dipping into python for every token is. If that isn't an issue I totally can!

Reply:
I love the focus on performance. I think we can keep the two different implementations under a single API function, with a fork in the logic depending on the params. i.e. the caller shouldn't have to think about that as a high level split, and we shouldn't have to duplicate all the documentation for the function, just the change in behavior based on those arguments.